addons/mail/static/src/models/message/message.js

```diff
@@ -612,6 +639,20 @@ function factory(dependencies) {
                 'originThreadIsModeratedByCurrentPartner',
             ],
         }),
+        /**
+         * States whether `threadname` and `subject` contain similar
+         * values except it's prefix.
+         *
+         * This is necessary to avoid displaying the subject, if
+         * the subject is same as threadname.
+         */
+        isSubjectEqualThreadName: attr({
```

Comment:
I would actually rename to avoid misleading, `isSubjectEqualThreadName` -> `isSubjectSimilarToOriginThreadName`
Because it is strange to have a field named "equal" when the values are not actually equal, and also it's important to specify origin thread is used.